storybookjs/storybook
# [addon-actions] Check result of getPropertyDescriptor for IE11

Issue: When using `action-addons` in IE11, it would throw "Unable to get property 'configurable' of undefined or null reference" because apparently IE returns `undefined` from `Object.getPropertyDescriptor` when called on a function.

## What I did

Check result of `Object.getPropertyDescriptor` before checking `.configurable`.

## How to test

Load storybook that uses `actions` in IE11 and verify that the iframe loads.

Is this testable with jest or storyshots? No, this is IE-specific behaviour.

Does this need a new example in the kitchen sink apps? No

Does this need an update to the documentation? No
